### PR TITLE
Add unsigned short type

### PR DESCRIPTION
**Pull request by nadongguri**
This patch adds unsigned short type.

ISSUE=#159

### CODE REVIEW

File: test/test_interface.idl [hunk 1]
```diff
@@ -32,6 +32,7 @@ interface TestInterface {
   byte byteMethod(byte number);
   octet octetMethod(octet number);
   short shortMethod(short number);
+  unsignedShort unsignedShortMethod(unsignedShort number);
```
Comment (romandev) [inline test/test_interface.idl]: Return type and parameter type should be ```unsigned short```.
Also, please rebase this patch.

Comment (nadongguri) [reply test/test_interface.idl]: @romandev, if I add spacebar in "unsigned short", build is failed in test_interface_bridge.cc
Release/obj/gen/test/test_interface_bridge.cc: In member function ‘Napi::Value TestInterfaceBridge::UnsignedShortMethod(const Napi::CallbackInfo&)’:
Release/obj/gen/test/test_interface_bridge.cc:438:32: error: ‘IDLUnsigned’ was not declared in this scope
 auto number = NativeTypeTraits<IDLUnsigned short>::NativeValue(info.Env(), info[0]);
                                ^
Release/obj/gen/test/test_interface_bridge.cc:438:49: error: template argument 1 is invalid
 auto number = NativeTypeTraits<IDLUnsigned short>::NativeValue(info.Env(), info[0]);
                                                 ^
bacardi.target.mk:122: recipe for target 'Release/obj.target/bacardi/gen/test/test_interface_bridge.o' failed

I will find out and fix it.

**Comment by romandev** [comment]
@nadongguri  FYI, the camelcase(pascalcase) issue was resolved. #168
Could you please rebase this patch and retry to fix again?

**Comment by nadongguri** [comment]
@romandev, okay.

**Comment by nadongguri** [comment]
@romandev, checked after rebase then using the type "unsigned short" was no problem to build and test. :D

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**